src/tests/modify.test.js

```diff
@@ -171,6 +208,44 @@ describe('modify() - basic mutations', () => {
     });
   });
 
+  it('replace fields that dont exist gets ignored', () => {
+    // IMPORTANT NOTE on this behavior:
+    // Context: At Remote we have a lot of global customization that run equally across multiple different JSON Schemas.
+    // With this, we avoid applying customizations to non-existing fields. (aka create fields)
+    // That's why we have the "create" config, specific to create new fields.
```

Comment:
Note: The create config will come in the other PR https://github.com/remoteoss/json-schema-form/pull/80